torch/csrc/api/include/torch/nn/modules/loss.h

```diff
@@ -52,7 +52,7 @@ TORCH_MODULE(L1Loss);
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ KLDivLoss ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-/// The `Kullback-Leibler divergence`_ Loss
+/// The Kullback-Leibler divergence loss measure
 /// See https://pytorch.org/docs/master/nn.html#torch.nn.KLDivLoss to learn
```

Comment:
The tag line of the C++ function ends up [in the documentation](https://pytorch.org/docs/master/nn.html#loss-functions) (in the table under that link). No need to use reference formatting, since the link is in the code and [the full docstring](https://pytorch.org/docs/master/generated/torch.nn.KLDivLoss.html#torch.nn.KLDivLoss) properly formats the link.